core-services/prow/02_config/_config.yaml

```diff
@@ -559,6 +559,7 @@ tide:
     openshift-knative/serverless-operator: squash
     openshift-knative/serving-operator: squash
     openshift-kni/baremetal-deploy: squash
+    openshift-metal3/dev-scripts: squash
```

Comment:
Not sure why this is showing up in the diff, it should already be in this file. Maybe PR needs rebasing?

Reply:
I just created this branch starting from yours